app/models/seance.rb

```diff
@@ -0,0 +1,28 @@
+class Seance < ApplicationRecord
+  belongs_to :hall
+  belongs_to :movie
+
+  before_validation :end_time
+
+  def end_time
+    self.end_time = self.start_time + self.movie.duration.minutes + 30.minutes
+  end
+
+  validate :is_used
+
+  def is_used
```

Comment:
It is considered a Ruby convention that if you have a simple method that returns a boolean, you follow the naming convention of adding an `?` to the end of it and omitting any `is/has` prefix.

```suggestion
  validate :used?

  def used?
```

You can read about it here:
[Ruby naming conventions](https://blog.codeminer42.com/and-understanding-one-of-rubys-coolest-naming-conventions-5a9300b75605/)